### DISCUSSION TITLE
Security Onion + Vsphere install error

### DISCUSSION THREAD

**Discussion by Slexin** [Security-Onion-Solutions/securityonion#13992] [Unsupported Versions] [answered]
I am trying to install Security Onion on Vsphere and I can not get pass the initial "chose this version of security onion you want to install".

I get the two following errors.

error: ../../grub-core/kern/disk_common.c:47:attempt to read or write of disk 'cd0'
error: ../../grub-core/loader/i386/efi/Linux.c:258:you need to load the kernel first

Any help would be much appreciated.

**Comment by dougburks** [comment, marked as answer]
Did you verify the ISO image as shown at https://github.com/Security-Onion-Solutions/securityonion/blob/2.4/main/DOWNLOAD_AND_VERIFY_ISO.md?

**Reply by Slexin** [reply]
I am able to get this far
![image](https://github.com/user-attachments/assets/098c2f83-3065-4139-aada-b280c38cdb00)
And then it gives me this page
![image](https://github.com/user-attachments/assets/3d1b0ca0-8d80-4dbd-87bf-e34866f0ad0d)
Based on this you think it might be corrupted? It is on a customer setup so I am having difficulties downloading a tool to verify the ISO.

**Reply by dougburks** [reply]
Yes, it is important that you verify the ISO image. Even if you can't verify the gpg signature, you should at least be able to verify the md5, sha1, or sha256 checksum using the utilities already available in most operating systems.

**Reply by Slexin** [reply]
You were right. The ISO File was being uploaded through Claroty where it was being corrupted somehow. Once it was uploaded to the vcenter without going through Claroty it loaded completely normal.
